src/homepage/homepage.html

```diff
@@ -67,7 +122,7 @@
   </section>
   <section id="partners">
     <h1>Premium Partner</h1>
-    <img src="images-optimized/sponsors/optiver_logo.svg" alt="Optiver logo" />
+    <img src="/images-optimized/sponsors/optiver_logo.svg" alt="Optiver logo" />
```

Comment:
Initially I said it should start with `/`, but I am second guessing my statement now.